### PR TITLE
[Ftr] update readme

### LINKED ISSUE #399: [ftr] add mission statement to Readme
This is not a technical issue, but rather to improve the documentation.

When you hit the "landing page" of the Edirom Online repo at Github, the Readme is automatically displayed (by GitHub). So, this is the very first information that users will get and thus should contain more details and context regarding the project. Here are some proposals:

* the first paragraph should be a short mission statement: overall goals, field of action, organizational background, visions, etc.
* Next, there should be show cases with images and pointers to existing edirom online editions, e.g. [Bargheer](https://bargheer.edirom.de/index.html), [Webers Klarinettenquintett](https://klarinettenquintett.weber-gesamtausgabe.de/)

… there might be more

### PR DESCRIPTION
**Pull request by krHERO**
this PR adds a lot of information, eg mission statement, project examples and aims to improve the structure of this document. the small description part how to test was moved to CONTRIBUTING (#408 ), since the whole description of contribution is living in this file. 
the paragraph "Starting an Edirom instance locally" was written by @daniel-jettka, thanks to this great improvement to help others to get their edirom started with this guidance!
since Peter started the issue to work on this file, i would be very happy, if @peterstadler is available for a review but of course everyone else willing to help with a review is invited as well.

Closes #399

### CODE REVIEW

**Review by peterstadler — COMMENTED** [review]
That is a really great improvement to the Readme. I left some comments which I hope are helpful!

File: README.md [hunk 1]
```diff
@@ -1,12 +1,58 @@
...
+The Edirom idea was born in 2004 at [Musikwissenschaftliches Seminar Detmold/Paderborn] and even after several years of Edirom development, the success of Edirom based on the same core concepts as in the beginning continues with numerous projects using and developing Edirom tools and creating digital musical editions with this software. Edirom tools were originally developed by the project [Entwicklung von Werkzeugen für digitale Formen wissenschaftlich-kritischer Musikeditionen] (2006–2012) funded by the DFG. The development of Edirom is now maintained as a community effort while being strongly supported and accompanied by [Virtueller Forschungsverbund Edirom] (ViFE), primarily based at the [Paderborn University]. ViFE aims to provide tools for scholars working with digital texts and music, especially those involved in editing historical documents.
+
+## Show cases
+
+To get some practical insights, look at this projects and editions that already use Edirom-Online.
+
+**Webers Klarinettenquintett op.34**
```
[the remark this answers is not in the document]
Comment (krHERO) [reply README.md]: thanks, i just added a [commit](https://github.com/Edirom/Edirom-Online/pull/410/commits/76fb190eea40dca480222b72b879f364d1d2e92d) to address your ideas. i ensured to provide all "institutions" with the links that are named here already in the introduction.

File: README.md [hunk 2]
```diff
@@ -20,39 +66,101 @@ If the submodules are not yet present after cloning, you can update them with:
...
+Alternatively we recommend to use a container image for building, e.g. [bwbohl/sencha-cmd]
 
 ```bash
 docker run --rm -it -v /ABSOLUTE/PATH/TO/YOUR/LOCAL/EDIROM-ONLINE/CLONE:/app --name ediBuild ghcr.io/bwbohl/sencha-cmd:latest
 ```
 
-When you have your system preapared with all Sencha Cmd prerequisites or you have your docker container running you have to execute a sencha build command through calling the build script included in this repository with one of the sencha build-type options (please refer to [sencha app build reference](https://docs.sencha.com/cmd/guides/advanced_cmd/cmd_reference.html#advanced_cmd-_-cmd_reference_-_sencha_app_build) for details), either in your native shell or in the container shell, e.g.:
+When you have your system preapared with all Sencha Cmd prerequisites or you have your docker container running you have to execute a sencha build command through calling the build script included in this repository with one of the sencha build-type options (please refer to [sencha app build reference] for details), either in your native shell or in the container shell, e.g.:
```
[the remark this answers is not in the document]
Comment (krHERO) [reply README.md:79]: feel free to shorten or split the sentences as you think it supports the reading flow :)

**Comment by krHERO** [comment]
hi @peterstadler i appreciate your review and accepted all of your ideas, so feel free to check the file again and merge if already ok for you. thanks a lot!

File: README.md [hunk 3]
```diff
@@ -1,12 +1,58 @@
...
+[License](https://github.com/Edirom/Edirom-Online#-License)**
 
-Edirom Online is a web application written in XQuery and JavaScript, and designed for deployment in [eXist-db](https://exist-db.org/). It is based on the work of the [_Edirom_-Project](https://edirom.de/edirom-projekt/) that originally was funded by the German Research Foundation (DFG). This software brings paperbased historio-critical editions of music texts to the web.
+</div>
 
-The software is still under high development and has to be seen as beta software.
+# Edirom-Online
 
-## Cloning this repository
+Edirom-Online is software for the **presentation and analysis of critical musical editions** in a digital format, particularly in the fields of musicology and philology. Edirom-Online supports various data formats commonly used in digital humanities, such as [TEI] (Text Encoding Initiative) for textual data and [MEI] (Music Encoding Initiative) for musical data. This allows the integration of different data formats, starting in the early days with texts, images and music and adding audio and even film within a single edition.
```
[the remark this answers is not in the document]
Comment (peterstadler) [reply README.md]: was this suggestion left out on purpose? Just to make sure it was not forgotten …

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (peterstadler) [reply README.md:79]: How about this?
```suggestion
When you have your system prepared with all Sencha Cmd prerequisites or you have your docker container running you are now set up to execute the sencha build command. Do this by calling the build script included in this repository with one of the sencha build-type options (please refer to [sencha app build reference] for details), either in your native shell or in the container shell, e.g.:
```

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (krHERO) [reply README.md]: sorry, i added a commit after your suggestions and (adressing one aspect of your review) but in this area, appearently i did not already accepted your suggestions. github does not allow me to commit your suggeestion siincee it is marked "outdated". sorry for the confusion witth my "dazwischen-grätschen"-commit. i accept all of your suggestions. please just add them all to the code.

File: README.md [hunk 4]
```diff
@@ -1,14 +1,60 @@
...
+[License](https://github.com/Edirom/Edirom-Online#-License)**
 
-Edirom Online is a web application written in XQuery and JavaScript, and designed for deployment in [eXist-db](https://exist-db.org/). It is based on the work of the [_Edirom_-Project](https://edirom.de/edirom-projekt/) that originally was funded by the German Research Foundation (DFG). This software brings paperbased historio-critical editions of music texts to the web.
+</div>
 
-The software is still under high development and has to be seen as beta software.
+# Edirom-Online
 
-## Cloning this repository
+Edirom-Online is software for the **presentation and analysis of critical musical editions** in a digital format, particularly in the fields of musicology and philology. Edirom-Online supports various data formats commonly used in digital humanities, such as [TEI] (Text Encoding Initiative) for textual data and [MEI] (Music Encoding Initiative) for musical data, that is visualized with [Verovio]. This allows the integration of different data formats, starting in the early days with texts, images and music and adding audio and even film within a single edition.
```
Comment (peterstadler) [inline README.md:18]: once more
 ```suggestion
Edirom-Online is a software for the **presentation and analysis of critical musical editions** in a digital format, particularly in the fields of musicology and philology. Edirom-Online supports various data formats commonly used in digital humanities, such as [TEI] (Text Encoding Initiative) for textual data and [MEI] (Music Encoding Initiative) for musical data, that is visualized with [Verovio]. This allows for the integration of different data formats, starting in the early days with texts, images and music and adding audio and even film within a single edition.  
```

**Review by peterstadler — APPROVED** [review]
Like it. That's a great improvement!

**Pull request merged**